deepspeech_wrap.h

```diff
@@ -0,0 +1,22 @@
+#ifdef __cplusplus
```

Comment:
Is there a reason why you renamed the file to `deepspeech_wrap` ?

Reply:
it was colliding with our `deepspeech.h`, renaming ensures people cannot hit that and can build with just adjusting `CFLAGS` and `LDFLAGS`